src/libs/ReportUtils.js

```diff
@@ -4168,6 +4168,14 @@ function getRoom(type, policyID) {
     return room;
 }
 
+/**
+ *
+ * @returns {Object}
+ */
+function getAllPersonalDetails() {
+    return allPersonalDetails;
+}
```

Comment:
I'm not sure about this - wouldn't be better to connect to onyx inside the `GroupChatUtils` file to retrieve this value directly?

Reply:
just updated